make_beam/get_delays_small.c

```diff
@@ -485,8 +490,20 @@ void get_delays(
                         // Strictly speaking, the condition (ch == 0) above is redundant, as the dipole configuration
                         // array takes care of that implicitly, but I'll leave it here so that the above argument
                         // is "explicit" in the code.
-                        jones[config_idx] = calc_jones( beam, az, PAL__DPIBY2-el, frequency + mi->chan_width/2,
-                                (unsigned int*)mi->delays[row], mi->amps[row], zenith_norm );
+                        
+                        // **NOTE** For newer versions of hyperbeam (i.e., >v0.2 or so) we need to provide additional 
+                        // arguments including the buffer for the Jones matrix data to be stored
+                        errInt = calc_jones( beam, az, PAL__DPIBY2-el, frequency + mi->chan_width/2,
+                                (unsigned int*)mi->delays[row], mi->amps[row], zenith_norm,
+                                NULL, // stand-in for array latitude
+                                0, // use IAU ordering (0 = don't)
+                                (double*)jones[config_idx] );
+                        
+                        if (errInt != 0)
+                        {             
+                            printf("Error encountered when computing beam Jones matrix via hyperbeam (line: %d, code: %d)\n" % (errInt, 495));
```

Comment:
Is this string formatting truly C? I didn't know the % operator worked like that. I think it should be
```
printf("Error encountered when computing beam Jones matrix via hyperbeam (line: %d, code: %d)\n", errInt, 495);
```

Reply:
Oh actually that was entirely my Python brain taking over :sweat_smile: 

You're right.